marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -168,7 +168,9 @@ const Browse: React.FC<Props> = ({ location }) => {
             location.state['query'])
         location.state['tableView'] ? toggleTableView(true) : toggleTableView(false);
     }
-
+    if(location.state && location.state['entityName'] && location.state['jobId']){
```

Comment:
Isn't this the same thing as line 178?

Reply:
@brucean52 if I have both entityName and jobId then only I am setting latestJobFacet, In the other scenario we are only checking for jobId, if that condition can be changed to include entityName too then i can remove this condition.

Reply:
Yeah, it can be changed in the future.